source/main.css

```diff
@@ -1,9 +1,141 @@
-/* This is just a test CSS file. */
-html,
-body {
-    height: 100%;
+
+.info {
+    margin: 0 auto;
+    width: 80%;
+}
+
+div p {
+    text-align: left;
+    color: #000000;
+    font-size: 14px;
+    font-family: helvetica;
+}
+
+.photo-1 {
```

Comment:
Let's change this class to `hero-image` … kinda borrowing a convention from Bootstrap.